tools/index_qdrant.sh

```diff
@@ -11,6 +11,10 @@ SCRIPT_PATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
 ROOT_PATH=$SCRIPT_PATH/..
 
 
+python -m code_search.index.files_to_json
```

Comment:
where the original files are coming from?

Reply:
https://github.com/qdrant/demo-code-search/blob/f93eab28ecdff42f63c8e7b0dde616cc8fc5aeaf/code_search/index/files_to_json.py#L28